awscli/examples/kms/decrypt.rst

```diff
@@ -1,41 +1,56 @@
+* Request the plaintext output as a text value.
 
-    The command assumes the ciphertext in ``ExampleEncryptedFile`` is binary data. The `encrypt examples <encrypt.html#examples>`_ demonstrate how to save a ciphertext this way.
+    The ``--query`` parameter tells the CLI to get only the value of the ``Plaintext`` field from the output. The ``--output`` parameter returns the output as text. 
 
-#. Uses the ``--output`` and ``--query`` parameters to control the command's output.
+* Base64-decode the plaintext and save it in a file.
 
-    These parameters extract the decrypted data, called the *plaintext*, from the command's output. For more information about controlling output, see `Controlling Command Output <https://docs.aws.amazon.com/cli/latest/userguide/controlling-output.html>`_ in the *AWS Command Line Interface User Guide*.
+    The  following example pipes (|) the value of the ``Plaintext`` parameter to the Base64 utility, which decodes it. Then, it redirects (>) the decoded output to the ``ExamplePlaintext`` file. 
 
-#. Uses the ``base64`` utility.
+Before running this command, replace the example key ID with a valid key ID from your AWS account. ::
 
-    This utility decodes the extracted plaintext to binary data. The plaintext that is returned by a successful ``decrypt`` command is base64-encoded text. You must decode this text to obtain the original plaintext.
+    aws kms decrypt \
+        --ciphertext-blob fileb://ExampleEncryptedFile \
+        --key-id 1234abcd-12ab-34cd-56ef-1234567890ab \
+        --output text \
+        --query Plaintext | base64 \
+        --decode > ExamplePlaintextFile
 
-#. Saves the binary plaintext to a file.
+This command produces no output. The output from the ``decrypt`` command is base64-decoded and saved in a file.
 
-    The final part of the command (``> ExamplePlaintextFile``) saves the binary plaintext data to a file.
+For more information, see `Decrypt <https://docs.aws.amazon.com/kms/latest/APIReference/API_Decrypt.html>`__ in the *AWS Key Management Service API Reference*.
 
-**Example 2: Using the AWS CLI to decrypt data from the Windows command prompt**
+**Example 2: To decrypt an encrypted message with a symmetric CMK (Windows command prompt)**
 
-The preceding example assumes the ``base64`` utility is available, which is commonly the case on Linux and Mac OS X. For the Windows command prompt, use ``certutil`` instead of ``base64``. This requires two commands, as shown in the following examples. ::
+The following example is the same as the previous one except that it uses the ``certutil`` utility to Base64-decode the plaintext data. This procedure requires two commands, as shown in the following examples. 
 
-    aws kms decrypt \
-        --ciphertext-blob fileb://ExampleEncryptedFile \
-        --output text \
+Before running this command, replace the example key ID with a valid key ID from your AWS account. ::
+
+    aws kms decrypt ^
```

Comment:
I'm not sure if `^` character is appropriate here? I'd assume we would want the `\` instead for line break

Reply:
Talked offline the characters are for command prompt. I did not see the title of the example. We can ignore my comment.